plugin/pkg/admission/namespace/lifecycle/admission.go

```diff
@@ -53,7 +53,7 @@ func (l *lifecycle) Admit(a admission.Attributes) (err error) {
 
 	// prevent deletion of immortal namespaces
 	if a.GetOperation() == admission.Delete && a.GetKind() == "Namespace" && l.immortalNamespaces.Has(a.GetName()) {
-		return errors.NewForbidden(a.GetKind(), a.GetName(), fmt.Errorf("namespace can never be deleted"))
+		return errors.NewForbidden(a.GetKind(), a.GetName(), fmt.Errorf("an immortal namespace can never be deleted"))
```

Comment:
@thockin - i do not want to expose the concept of immortal to an end-user.

I am good with "namespace may not be deleted"